app/views/course/assessment/submission/submissions/_guided.html.slim

```diff
@@ -1,16 +1,32 @@
-nav
-  ul.pagination
-    - (1..@assessment.questions.length).each do |i|
-      li class="#{guided_nav_class(i)}"
-        = link_to i, [:edit, current_course, @assessment, @submission, step: i], class: ['btn', 'btn-default']
+- if @submission.attempting?
```

Comment:
Unless unsubmit ? because all the questions/answers will be displayed at the same page (same as V1), it's a overview page.